packages/pages/src/dev/server/middleware/indexPage.ts

```diff
@@ -225,11 +225,18 @@ const createEntityPageListItems = (
     )}/${entityId}`;
   };
 
+  const formatDisplayValue = (entityId: string, slug: string | undefined) => {
+    if (!slug) {
+      return entityId;
+    }
+    return `${slug}`;
```

Comment:
I don't think this needs quotes

Reply:
Just return slug

Reply:
Better yet, this function doesn't seem necessary. Just use `slug ?? entityId` where it's used

Reply:
Agree, done